app/views/layouts/darkswarm.html.haml

```diff
@@ -36,10 +36,6 @@
     = action_cable_meta_tag
 
   %body{ class: body_classes, "body-scroll": "true", "data-turbo": "false" }
-    / [if lte IE 8]
-      = render partial: "shared/ie_warning"
-      = javascript_include_tag "iehack"
```

Comment:
We can remove that JS file, too. It's referenced in config/application.rb and located in app/assets/javascripts/iehack.js.